Lib/test/test_venv.py

```diff
@@ -156,14 +157,16 @@ def test_config_file_command_key(self):
             ('upgrade', '--upgrade'),
             ('upgrade_deps', '--upgrade-deps'),
             ('prompt', '--prompt'),
+            ('gitignore', '--without-gitignore'),
         ]
+        negated_attrs = {'with_pip', 'symlinks', 'gitignore'}
         for attr, opt in attrs:
             rmtree(self.env_dir)
             if not attr:
                 b = venv.EnvBuilder()
             else:
                 b = venv.EnvBuilder(
-                    **{attr: False if attr in ('with_pip', 'symlinks') else True})
+                    **{attr: False if attr in negated_attrs else True})
```

Comment:
Perhaps this, though may make sense to retain the ternary for explitness:
```suggestion
                    **{attr: attr not in negated_attrs})
```